src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -160,15 +181,15 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
   };
 
   private handleLabelFilter(key: string, value: string, frame: DataFrame | undefined, operator: FilterType) {
-    const labelType = frame ? getLabelTypeFromFrame(key, frame) : LabelType.Parsed;
-    const variableName = getFilterTypeFromLabelType(labelType, key, value);
-    addToFilters(key, value, operator, this, variableName);
+    const variableType = getDesiredVariableForLabel(frame, key, this);
```

Comment:
```suggestion
    const variableType = getVariableForLabel(frame, key, this);
```
Super minor, but not a big fan of these "desired".